### ISSUE TITLE
Inquiry Regarding Results Discrepancy in Ex1 Flat_tension_PDE_based_NN Section

### ISSUE THREAD

**Issue by Jiarui-Shi-DUT** [weili101/Deep_Plates#7]
Dear Wei Li,

Thank you for providing the code for your paper, which has played a very important role in helping me understand the numerical method PINN. I have a few questions I would like to ask you. In your section Ex1 Flat_tension_PDE_based_NN, I followed the code you provided for calculations. Based on Figure 6 in your paper, I adjusted the epoch to 12,000 iterations, and set the loss function as loss = 10.0*loss1 + loss2 + loss3. However, my results show that the maximum horizontal displacement is 0.1144, which differs somewhat from the result shown in Figure 7(c) of your paper. Is this difference due to the fact that you used a larger number of epochs, or is it caused by the fluctuation in results due to the random initialization of the neural network parameters? As a beginner, I kindly ask for your guidance.

![Image](https://github.com/user-attachments/assets/9ea5a04d-9ae6-463f-bf55-7742372bb31d)

**Comment by Jiarui-Shi-DUT** [comment]
It really seems to be caused by fluctuations in the results caused by random initialization of the neural network parameters.

![Image](https://github.com/user-attachments/assets/7511db35-1d2e-4453-a69a-e9d042514228)

**Comment by weili101** [comment]
Thank you for trying out the code. Yes, there would be variations in predictions due to random initialization.
